.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,6 +20,17 @@ For end-user related support questions, please refer to one of the following:
 
 Bug
 
+### Did you try using a 1.7.x configuration for the version 2.0?
+
+- [ ] Yes
+- [ ] No
+
+<!--
+
+If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.

```

Comment:
```suggestion
If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```

Reply:
I'd suggest also replacing `configs` with `configuration` for consistency